build_msvc/common.qt.vcxproj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<PropertyGroup><BuildDependsOn>$(BuildDependsOn);CopyBuildArtifacts</BuildDependsOn></PropertyGroup>
+  <Target Name="CopyBuildArtifacts" Condition="'$(ConfigurationType)' != 'StaticLibrary'">
+    <ItemGroup>
+      <BuildArtifacts Include="$(OutDir)$(TargetName)$(TargetExt)"></BuildArtifacts>
+      <BuildArtifacts Include="$(OutDir)$(TargetName).pdb" Condition="Exists('$(OutDir)$(TargetName).pdb')"></BuildArtifacts>
+    </ItemGroup>
+    <Copy SourceFiles="@(BuildArtifacts)" SkipUnchangedFiles="true" DestinationFolder="..\..\src\qt\" Condition="'$(OutDir)' != ''"></Copy>
+  </Target>
+  <Import Project="common.qt.vcxproj.user" Condition="Exists('common.qt.vcxproj.user')" />
+</Project>
```

Comment:
Do the Qt exe's need to be copied to src\qt for any paricular reason? Is that where the test scripts look for them? I believe Nicolas originally added the `common.vcxproj` config to copy the exe's to the src directory for conveniece. Perhaps it's sufficient here as well and avoids the need for the new `common.qt.vcxproj` file?

Reply:
No test script uses `bitcoin-qt.exe`. The only reason is my confusion:

> Not finding `bitcoin-qt.exe` ... in the `src\qt\` folder could confuse users.

Probably, it is just a biased opinion.

Reply:
If the test scripts don't need it in `src\qt` then I would recommend just including `common.vcxproj` and not bothering with the new `common.qt.vcxproj` file. Windows devs will normally look in the default output directory of `build_msvc\x64\Release` directory.

Reply:
> Windows devs will normally look in the default output directory of `build_msvc\x64\Release` directory.

Agree. Going to close this.